alica_engine/src/engine/RunnableObject.cpp

```diff
@@ -50,7 +52,46 @@ void RunnableObject::start(RunningPlan* rp)
     }
     ++_signalState;
     _signalContext.store(rp);
-    _engine->editScheduler().schedule(std::bind(&RunnableObject::doInit, this));
+    if(_requiresParameters) {
+        assert(rp->getParent());
+        assert(rp->getParent()->getBasicPlan());
+        const auto& wrappers = rp->getParent()->getActiveState()->getConfAbstractPlanWrappers();
+        auto it = std::find_if(wrappers.begin(), wrappers.end(), [this](const auto& wrapper_ptr){
+            return wrapper_ptr->getAbstractPlan()->getName() == _name;
+        });
+        assert(it != wrappers.end());
+
+        int64_t wrapperId = (*it)->getId();
+
+        std::shared_ptr<BlackBoard> parentBlackBoard = rp->getParent()->getBasicPlan()->getBlackBoard();
+        auto& planAttachment = rp->getParent()->getBasicPlan()->getPlanAttachment(wrapperId);
+        auto initCall = [&](){
+            _blackBoard->clear();
```

Comment:
This will crash now because there is no default blackboard. After removing from constructor you need to allocate here.